openedx/core/djangolib/tests/test_markup.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
 """
-Tests for util.markup
+Tests for openedx.core.djangolib.markup
 """
 
 import unittest
 
 import ddt
+from mako.template import Template
```

Comment:
Why change this import? edxmako.template.Template is a different class than mako.template.Template.